include/hc.hpp

```diff
@@ -2778,17 +2778,19 @@ extern "C" inline unsigned int __activelanecount_u32_b1(unsigned int input) __HC
  * wavefront and return non-zero if and only if predicate evaluates to non-zero
  * for any of them.
  */
+extern "C" __attribute__((const)) bool __ockl_wfany_i32(int) __HC__;
```

Comment:
Drop the const attribute.

Reply:
I'm dropping the attribute from OCKL.  It isn;t correct for these uses of the function.